smartapps/samsung/sleepsense-in-bed.src/sleepsense-in-bed.groovy

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
***OVERALL COMMENT***

This functionality should be rolled up into the previous app. There's no reason that the user can't configure a listener for "in bed"/"out of bed" & "sleeping"/"not not sleeping".